### PR TITLE
Verify customer suspend state when tx

### PR DESCRIPTION
**Pull request by josepfo**

### CODE REVIEW

**Comment by josepfo** [comment]
["Suspend" state not showing on admin](https://trello.com/c/vee9n3T2/527-suspend-state-not-showing-on-admin)

**Review by chaotixkilla — APPROVED** [review]
Seems good, really like the JSDoc comments. Perhaps it is something we should be doing more

File: lib/brain.js [hunk 1]
```diff
@@ -1546,6 +1546,96 @@ Brain.prototype.twoWayMode = function twoWayMode () {
...
+ * one of the enabled compliance tier thresholds
+ *
+ * @name isSuspended
+ * @function
+ *
+ * @param {object} customer Acting customer
+ * @param {date} now Current date
+ * @returns {bool} Whether customer is suspended or not
+ */
+Brain.prototype.isSuspended = function isSuspended (customer, now) {
```
Comment (RafaelTaranto) [inline lib/brain.js:1562]: I don't see a reason why now is a param, can't we just create a new Date within this function?

File: lib/brain.js [hunk 2]
```diff
@@ -1546,6 +1546,96 @@ Brain.prototype.twoWayMode = function twoWayMode () {
...
+ * else the bill screen will be displayed with the relative error message
+ *
+ * @name showSuspendedCustomer
+ * @function
+ *
+ * @param {object} customer Acting customer
+ * @param {date} now Current date
+ *
+ */
+Brain.prototype.showSuspendedCustomer = function showSuspendedCustomer (customer, now) {
```
Comment (RafaelTaranto) [inline lib/brain.js:1579]: Same as above

**Review by RafaelTaranto — APPROVED** [review]

File: lib/brain.js [hunk 3]
```diff
@@ -1546,6 +1546,96 @@ Brain.prototype.twoWayMode = function twoWayMode () {
...
+* current date and the suspension time
+*
+* @name hardLimitReached
+* @function
+*
+* @param {object} customer Acting customer
+* @param {date} now Current date
+* @returns {object} hard limit
+*/
+Brain.prototype.hardLimitReached = function hardLimitReached (customer, now) {
```
Comment (RafaelTaranto) [inline lib/brain.js:1623]: Can we rename this? By the name looks like we're testing for a boolean. 

Maybe `getHardLimitReachedData`

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**